apps/fares/lib/format.ex

```diff
@@ -93,6 +93,8 @@ defmodule Fares.Format do
   def name(:ada_ride), do: "ADA Ride"
   def name(:premium_ride), do: "Premium Ride"
   def name(:invalid), do: "Invalid Fare"
+  def name(:massport_shuttle), do: "Massport Shuttle"
+  def name("Massport-" <> _id), do: "Massport Shuttle"
```

Comment:
question: Is `Massport Shuttle` a valid display name?

Reply:
It's a choice. We can certainly update this to add better names if we know them!